src/taxes/pages/TaxCountriesPage/TaxCountriesPage.tsx

```diff
@@ -0,0 +1,140 @@
+import {
+  Card,
+  CardContent,
+  InputAdornment,
+  TextField
+} from "@material-ui/core";
+import VerticalSpacer from "@saleor/apps/components/VerticalSpacer";
+import CardTitle from "@saleor/components/CardTitle";
+import Container from "@saleor/components/Container";
+import Grid from "@saleor/components/Grid";
+import PageHeader from "@saleor/components/PageHeader";
+import Skeleton from "@saleor/components/Skeleton";
+import {
+  CountryFragment,
+  TaxCountryConfigurationFragment
+} from "@saleor/graphql";
+import { sectionNames } from "@saleor/intl";
+import {
+  List,
+  ListHeader,
+  ListItem,
+  ListItemCell,
+  makeStyles,
+  PageTab,
+  PageTabs,
+  SearchIcon
+} from "@saleor/macaw-ui";
+import { parseQuery } from "@saleor/orders/components/OrderCustomerAddressesEditDialog/utils";
+import { taxesMessages } from "@saleor/taxes/messages";
+import React from "react";
+import { FormattedMessage, useIntl } from "react-intl";
+
+import TaxInput from "../../components/TaxInput";
+import TaxCountriesMenu from "./TaxCountriesMenu";
+
+interface TaxCountriesPageProps {
+  countryTaxesData: TaxCountryConfigurationFragment[];
+  countries: CountryFragment[];
+  selectedCountryId: string;
+  handleTabChange: (tab: string) => void;
+}
+
+const useStyles = makeStyles(
+  () => ({
+    inputPadding: {
+      padding: "16px 0 16px 0"
+    }
+  }),
+  { name: "TaxCountriesPage" }
+);
+
+export const TaxCountriesPage: React.FC<TaxCountriesPageProps> = props => {
+  const {
+    countryTaxesData,
+    countries,
+    selectedCountryId,
+    handleTabChange
+  } = props;
+
+  const intl = useIntl();
+  const classes = useStyles();
+
+  const [query, setQuery] = React.useState("");
+
+  const currentCountry = countryTaxesData?.find(
+    country => country.countryCode === selectedCountryId
+  );
+
+  const filteredRates = currentCountry?.taxClassCountryRates.filter(
+    rate => rate.taxClass.name.search(new RegExp(parseQuery(query), "i")) >= 0
+  );
+
+  return (
+    <Container>
+      <PageHeader title={intl.formatMessage(sectionNames.taxes)} />
+      <PageTabs value="countries" onChange={handleTabChange}>
+        <PageTab label={"Channels"} value="channels" />
+        <PageTab label={"Countries"} value="countries" />
+        <PageTab label={"Tax classes"} value="classes" />
```

Comment:
Untranslated labels - I think we should use `FormattedMessage` or intl func

Reply:
Yeah i missed that